File: dataset/output/OPENAI_gpt-4o/Task75_OPENAI_gpt-4o.java
import java.io.IOException;

public class KillProcess {

    public static void main(String[] args) {
        if (args.length != 1) {
            System.err.println("Usage: java KillProcess <pid>");
            System.exit(1);
        }

        String pid = args[0];
        try {
            Process p = Runtime.getRuntime().exec("kill -9 " + pid);
            p.waitFor();
            if (p.exitValue() == 0) {
                System.out.println("Process " + pid + " terminated.");
            } else {
                System.out.println("Failed to terminate process " + pid + ".");
            }
        } catch (IOException | InterruptedException e) {
            e.printStackTrace();
        }
    }
}

File: dataset/output/OPENAI_gpt-4o/Task75_OPENAI_gpt-4o.cpp
#include <iostream>
#include <csignal>
#include <cstdlib>

int main(int argc, char *argv[]) {
    if (argc != 2) {
        std::cout << "Usage: " << argv[0] << " <PID>" << std::endl;
        return 1;
    }
    int pid = std::atoi(argv[1]);
    if (kill(pid, SIGKILL) == 0) {
        std::cout << "Process " << pid << " terminated." << std::endl;
    } else {
        std::perror("Error");
    }
    return 0;
}